src/presentation/styles/typography.pcss

```diff
@@ -1,35 +1,74 @@
 :root {
   --text-heading-1 {
+      color: var(--color-text-heading);
       font-weight: 700;
       font-size: 2.75rem;
       line-height: 120%;
       letter-spacing: 0.1px;
   }
 
   --text-heading-2 {
+      color: var(--color-text-heading);
       font-weight: 700;
       line-height: 120%;
       font-size: 1.75rem;
       letter-spacing: 0.1px;
   }
 
   --text-heading-3 {
+      color: var(--color-text-heading);
       font-weight: 700;
       font-size: 1.375rem;
       line-height: 120%;
       letter-spacing: 0.1px;
   }
 
   --text-body {
+      color: var(--color-text-main);
       font-weight: 400;
       line-height: 25px;
       font-size: 1rem;
   }
 
   --text-small {
+      color: var(--color-text-main);
       font-weight: 400;
       font-size: 0.875rem;
       line-height: 120%;
       letter-spacing: 0.14px;
   }
+
+    .block {
+        display: flex;
+        flex-direction: column;
+        gap: var(--spacing-xs);
+    }
+
+    h1 {
+        @apply --text-heading-1;
+        padding: 0;
+        margin: 0 0 var(--spacing-ms);
+    }
+
+    h2 {
+        @apply --text-heading-2;
+        padding: 0;
+        margin: var(--spacing-mm) 0 var(--spacing-xxs);
+    }
+
+    h3 {
+        @apply --text-heading-3;
+        padding: 0;
+        margin: var(--spacing-xs) 0 var(--spacing-very-x);
+    }
+
+    .ce-paragraph {
```

Comment:
its better to place editor styles at the Editor component

Reply:
I'm not sure, but styling global styles inside components is a good pattern?

Reply:
I don't think it is a global style in that case. It's a style of the Editor. There won't be a `.ce-paragraph` outside the `<Editor>` component